actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -301,6 +336,21 @@ private void createUser(Request actorMessage) {
     processUserRequest(userMap, callerId);
   }
 
+  private void validateUserType(Map<String, Object> userMap, boolean isCustodianOrg) {
+    String userType = (String) userMap.get(JsonKey.USER_TYPE);
+    if (StringUtils.isNotBlank(userType)) {
+      if (userType.equalsIgnoreCase(UserType.TEACHER.getTypeName()) && isCustodianOrg) {
```

Comment:
collapse above 2 conditionals -> userType not blank and teacher and isCustodianOrg